Add information disable cross-browser support (#4138)
wso2/docs-is: en/identity-server/next/docs/guides/authentication/passwordless-login/add-passwordless-login-with-magic-link.md

{% set configure_email_sender = "- You need to [configure the email sending module](../../../deploy/configure-email-sending.md) for the WSO2 Identity Server to send the Magic link to the users." %}

> {% set disable_cross_browser_support = "If you want to remove the cross-browser functionality, you can do so by adding the following configuration in the `deployment.toml` file.

Reviewer: Please mention the location of the deployment.toml file. Also say what happens with this config.

For example: "Enabling this config forces users to open the magic link from the same browser from which they initiated the login process."

Author: Fixed with 5eb3acb1ce64db1871d0c170a1a8e7e2bc61ea99

<img width="1127" alt="Screenshot 2023-11-14 at 11 09 06 AM" src="https://github.com/wso2/docs-is/assets/32576163/aecad1fc-f81b-4341-8472-174cc08d6e3b">

{% set configure_email_sender = "- You need to [configure the email sending module](../../../deploy/configure-email-sending.md) for the WSO2 Identity Server to send the Magic link to the users." %}

> {% set disable_cross_browser_support = "If you want to remove the cross-browser functionality, you can do so by adding the following configuration in the `<IS_HOME>/repository/conf/deployment.toml` file.

Reviewer: ```suggestion
{% set disable_cross_browser_support = "If you want to remove the cross-browser functionality, you can do so by adding the following configuration in the `deployment.toml` file found in the {IS_HOME}/repository/conf/ directory.
```

        ```toml
           [session.nonce.cookie]
           default_whitelist_authenticators=[] 
        ```

>         This configuration will ensure that the magic link can only be opened in the same browser where the login was originally initiated.

Reviewer: ```suggestion
        This configuration ensures that the magic link can only be opened using the same browser from which the login was initiated.
```
